ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/transactions/TestNodeList.java

```diff
@@ -59,12 +59,6 @@ public TestNode create(
     return node;
   }
 
-  public void startNetworks() {
```

Comment:
Each individual network is already started [here](https://github.com/PegaSysEng/pantheon/blob/e95d0681450b12df36102d16c1b99c02ea2aca2f/ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/transactions/TestNode.java#L158)